src/Services/privateApiService.js

```diff
@@ -53,4 +53,16 @@ export const privateRequestPut = async(path,id,body) => {
     return data;
 };
 
-export default Get;
+
+const privatePost = async (url, data) => {
+    const authorizationHeader = getAuthorizationHeader();
+    if (!authorizationHeader.Authorization) return;
```

Comment:
you should return an error that the user is not authorized to execute the endpoint

Reply:
changes added